pb/ctrl_pb/ctrl.proto

```diff
@@ -73,6 +75,7 @@ message Route {
         string binding = 1;
         string address = 2;
         string destination = 3;
+        map<uint32, bytes> peerData = 4;
```

Comment:
Why is this `peerData` in some places and `hostData` in others? Would it make sense to just call ti `peerData` across the board, and ship it?

Reply:
`hostData` is provided by binding identity (hosting endpoint), `peerData` is send by connecting entity (client). Keeping them different makes a little more clear what's going on

Reply:
But from the perspective of the fabric, it's all the same thing... it's data that's being shuffled between pairs of endpoints? So that distinction kind of isn't really evident at the fabric layer?

Reply:
replaced